src/NetTopologySuite/Geometries/Implementation/CoordinateArraySequence.cs

```diff
@@ -64,7 +64,7 @@ public CoordinateArraySequence(Coordinate[] coordinates, int dimension, int meas
         {
             if (coordinates == null)
             {
-                Coordinates = new Coordinate[0];
+                Coordinates = Array.Empty<Coordinate>();
```

Comment:
:page_facing_up: Same basic note about behavior change as in `MinimumBoundingCircle.cs`